CHANGES.md

```diff
@@ -1,17 +1,43 @@
 ## DC/OS 1.11-dev
 
+```
+* For any significant improvement to DC/OS add an entry to Fixed and Improved section.
+* For Security updates, please call out in Security updates section.
+* Add to the top of the existing list.
+* External Projects like Mesos and Marathon shall provide a link to their published changelogs.
+
+Format of the entries must be.
+
+* Entry with no-newlines. (DCOS_OSS_JIRA)
+<new-line>
+* Entry two with no-newlines. (DCOS_OSS_JIRA_2)
```

Comment:
"Entry one without newline" ?
"Entry two without newline" ?

Reply:
The first line was Entry one.